prompt_toolkit/contrib/telnet/server.py

```diff
@@ -59,6 +61,12 @@ def _initialize_telnet(connection: socket.socket) -> None:
     # Negotiate window size
     connection.send(IAC + DO + NAWS)
 
+    # Negotiate terminal type
```

Comment:
Both sequences here have the comment "Negotiate terminal type".
Are they doing the same thing?

Reply:
The full exchange for selecting the first terminal type supported by the client (which is generally the best type the client supports) is actually:
```
[Server]> IAC DO TERMINAL-TYPE
[Client]< IAC WILL TERMINAL-TYPE
[Server]> IAC SB TERMINAL-TYPE SEND IAC SE
[Client]< IAC SB TERMINAL-TYPE IS "VT100" IAC SE
```
Source: http://mud-dev.wikidot.com/telnet:negotiation

Here, I don't wait for `IAC WILL TERMINAL-TYPE` because I didn't want those changes to get too complicated as it seems safe to assume that the client will accept the terminal type negociation.

Is adding a comment enough or do you want me to implement the negotiation more rigorously?

Reply:
A comment is enough. We can always improve the implementation if needed for somebody.